templates/components/common/alert/alert-modal.html

```diff
@@ -1,7 +1,19 @@
 <div id="alert-modal" class="modal modal--alert modal--small" data-reveal data-prevent-quick-search-close>
-    <div class="swal2-icon swal2-error swal2-icon-show"><span class="swal2-x-mark"><span class="swal2-x-mark-line-left"></span><span class="swal2-x-mark-line-right"></span></span></div>
+    <div class="alert-icon error-icon">
+        <span class="icon-content">
+            <span class="line line-left"></span>
+            <span class="line line-right"></span>
+        </span>
+    </div>
+
+    <div class="alert-icon warning-icon">
+        <div class="icon-content">!</div>
+    </div>
 
     <div class="modal-content"></div>
 
-    <div class="button-container"><button type="button" class="confirm button" data-reveal-close>OK</button></div>
+    <div class="button-container">
+        <button type="button" class="confirm button" data-reveal-close>OK</button>
+        <button type="button" class="cancel button" data-reveal-close>Cancel</button>
```

Comment:
"OK" & "Cancel" should be translatable. We have a _common_ section in json lang files that can be used for this purpose. Or other sections that you find more suitable.

Reply:
@bc-alexsaiannyi done!